mne/io/reference.py

```diff
@@ -281,7 +281,7 @@ def set_bipolar_reference(inst, anode, cathode, ch_name=None, ch_info=None,
 
     Parameters
     ----------
-    inst : instance of Raw | Epochs |Evoked
+    inst : instance of Raw | Epochs | Evoked
```

Comment:
yeah, it's just that I still think of `Raw` as a parent class that the other classes are based on. now it's `_BaseRaw` which makes sense. and the `Raw` is just `RawFIF` but named that way for historical reasons.